rest/tls/README.md

```diff
@@ -136,11 +176,29 @@ that it is as you expected.
 
 You are looking for something that looks like:
 ```
-"configuration":{"configurationset":[{"configuration":{"name":"selfsign","ca":
+"configuration":{"configurationset":[{"configuration":{"name":"myserver","ca":
 "-----BEGIN CERTIFICATE-----\nMIIF5zCCA8+gAwIBAgIUS2BwtghuA7PREQ5AWzOeeT+tCe4wDQYJKoZIhvcNAQEL\nBQAwbTE...
 ```
 notice the '\n'.  If you see spaces you've done something wrong upsteram.
 
+### Reserved environment variables
+
+`STEPZEN_SERVER_URL` is used by the stepzen cli to identify the API Connect for GraphQL server and should be avoided to prevent stepzen cli issues.
+
+Here's a full list of stepzen cli special variables you should avoid using.
+```
+STEPZEN_ENABLE_PERF
+STEPZEN_DISABLE_ANALYTICS
+STEPZEN_INTERACTIVE
+STEPZEN_CONFIG_FILE
+STEPZEN_DOCKER_IMAGE
+STEPZEN_DOCKER_TAG
+STEPZEN_DEPLOYMENT_TYPE
+STEPZEN_ZENCTL_API_URL
+STEPZEN_JSON2SDL_SERVER_URL
+STEPZEN_CONFIG_CONTENT
+STEPZEN_SEGMENT_PRODUCT
+```
```

Comment:
Not helpful enough here to include and distracting.

```suggestion
Here's a full list of stepzen cli special variables you should avoid using.
```
STEPZEN_ENABLE_PERF
STEPZEN_DISABLE_ANALYTICS
STEPZEN_INTERACTIVE
STEPZEN_CONFIG_FILE
STEPZEN_DOCKER_IMAGE
STEPZEN_DOCKER_TAG
STEPZEN_DEPLOYMENT_TYPE
STEPZEN_ZENCTL_API_URL
STEPZEN_JSON2SDL_SERVER_URL
STEPZEN_CONFIG_CONTENT
STEPZEN_SEGMENT_PRODUCT
```
```